packages/hash/api/src/db/postgres/link/sql/link_versions.util.ts

```diff
@@ -111,17 +112,19 @@ export const updateLinkVersionIndices = async (
       index = index + ${operation === "increment" ? 1 : -1},
       updated_at = ${params.updatedAt.toISOString()},
       updated_by_account_id = ${params.updatedByAccountId}
+    from
+      links
     where ${sql.join(
       [
-        sql`source_account_id = ${params.sourceAccountId}`,
-        sql`source_entity_id = ${params.sourceEntityId}`,
-        sql`path = ${params.path}`,
-        sql`and index is not null`,
+        sql`links.source_account_id = ${params.sourceAccountId}`,
+        sql`links.source_entity_id = ${params.sourceEntityId}`,
```

Comment:
The only reason I started destructuring `params` was for the `minimumIndex` and `maximumIndex` which are reused more than once. Would you be okay with me just destructuring them, and using `params.operation` instead of destructuring it also?

I don't like the idea of destructuring everything here because it'll take up 8 lines of code to do it.

Reply:
Have done this in https://github.com/hashintel/hash/pull/403/commits/976f81afb854c548bff0fdca5b1da7af8eb6a01a, let me know if this addresses your nit

Reply:
Ah okay, so in cases where we just use the parameters once, we just refer to them directly instead of destructuring? I think that's fair to keep doing.

Reply:
Yeah the change makes sense :)

Reply:
I think this is one of the things that's up to the developer, I prefer this:
```TypeScript
        typeof params.maximumIndex !== "undefined"
          ? sql`link_versions.index <= ${params.maximumIndex}`
          : [],
```
over this:
```Typescript
        typeof params.maximumIndex !== "undefined"
          ? sql`link_versions.index <= ${params.maximumIndex}`
          : [],
```